packages/core/src/client.ts

```diff
@@ -2154,6 +2155,30 @@ export class MedplumClient extends EventTarget {
     });
   }
 
+  /**
+   * Kick off system level bulk export
+   */
+  async bulkExport(resources: string): Promise<Partial<BulkDataExport>> {
+    const url = this.fhirUrl('$export');
```

Comment:
This assumes that we're only doing system level export.  I think we need to support specifying a base export.

For example, consider the different exports:

- https://bcda.cms.gov/guide.html#try-the-api
  - `https://sandbox.bcda.cms.gov/api/v1/Patient/$export`
- https://docs.google.com/presentation/d/14ZHmam9hwz6-SsCG1YqUIQnJ56bvSqEatebltgEVR6c/edit?usp=sharing
  - `[FHIR Server Base]/Patient/$export`
  - `[FHIR Server Base]/Group/[group id]/$export`
  - `[FHIR Server Base]/$export`

Maybe `baseExport`?  And we could even make it an optional parameter.

Can we also modify `resources: string` to (1) be optional and (2) be aligned with the `_type` param?  So, maybe "types" or "resourceType"?

I.e.,

```ts
async bulkExport(baseExport = '', resourceTypes?: string): Promise<Partial<BulkDataExport>> {
```